boards/arm/nrf5340_audio_dk_nrf5340/dts/bindings/nrf5340_audio_dk.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  This file contains specific pin mappings for the nRF5340 Audio DK.
+
+compatible: "nrf5340_audio_dk"
+
+include: base.yaml
+
+properties:
+    hw-codec-in-gpios:
+      type: phandle-array
+      required: true
+      description: GPIO from HW codec
+
+    hw-codec-irq-in-gpios:
+      type: phandle-array
+      required: true
+      description: Interrupt from HW codec
+
+    hw-codec-reset-out-gpios:
+      type: phandle-array
+      required: true
+      description: Reset HW codec
+
+    hw-codec-sel-out-gpios:
+      type: phandle-array
+      required: true
+      description: HW codec selector
```

Comment:
this is still not using bindings correctly. Since this is something not needed for upstream, we can add overlays in NCS, so the upstream board doesn't have to deal with hw-codec stuff.

Reply:
Even if the cirrus logic drivers are not upstreamed (yet) - it has value for the upstreamed board definitions to include info on where the HW Codec is hooked up. The board *does* include the HW Codec and it is one of the key values over e.g. nRF5340 DK.

Reply:
Then bindings need to be designed correctly.

Reply:
After discussing with @gmarull, this bindings file has been removed.